### DISCUSSION TITLE
modify GitTree parameters

### DISCUSSION THREAD

**Discussion by oscarBarEs** [Kitware/trame#502] [General] [answered]
Hi! On my project there's a GitTree but also there's a switch to change the theme of the app.

```
      vuetify.VSwitch(
          v_model="$vuetify.theme.dark",
          hide_details=True,
          dense=True,
      )
```

When the theme is white you can perfectly read the GitTree due to the text being black, but when u switch it, all text from tha app changes colours except the Tree. 

The [documentation](https://trame.readthedocs.io/en/latest/trame.widgets.trame.html#trame.widgets.trame.GitTree) shows that it has a parameter called  text_color , so i am trying to modify it, but its not changing anything

```
      trame.GitTree(
          sources=(
              "pipeline",
              self.sources,
          ),
          actives_change=(self.actives_change, "[$event]"),
          text_color='#FFFFFF'  # set the text color to white

      )
```
So, how can I acces the Vuetify styling attributes?

**Comment by jourdain** [comment]
`text_color=("$vuetify.theme.dark ? '#fff' : '#ccc'",),`

**Reply by jourdain** [reply, marked as answer]
Here is a full example https://github.com/Kitware/conceptual-modeler/blob/main/conceptual_modeler/app/ui.py#L280-L291

**Reply by oscarBarEs** [reply]
The example you linked works perfectly, the other not.
Maybe its because the text_color property has to be an array.

And thanks a lot (again) :)

**Reply by jourdain** [reply]
yes that was because it was expecting an array and I just extended your code (assuming it was working).
The array is to handle regular text vs the active one which revert the color contrast.
